steps/github-step-repositories-describe/spec.schema.json

```diff
@@ -0,0 +1,35 @@
+{
```

Comment:
This file could use a formatting/indentation pass. For JSON usually just feeding it through `jq` will get you a good result if you don't have a formatter installed in your editor.

Reply:
Would you link to https://pygithub.readthedocs.io/en/latest/github_objects/Organization.html#github.Organization.Organization.get_repos directly in the schema or in the README?

Reply:
You could put a link in the README if you like. It's sometimes helpful to know what API calls a particular step maps to.

Reply:
I would actually just link to https://docs.github.com/en/rest/repos/repos#list-organization-repositories though as step users don't usually care what language the step is implemented in.